netlify.toml

```diff
@@ -3,11 +3,6 @@
 [build]
   publish = "public"
 
-[dev]
-  publish = "public"
-  port = 64946
-  autolaunch = true
-
```

Comment:
Also removing this since we don't really need `netlify dev` for now. Will add it back if we want it later.